include/SFML/Window/Keyboard.hpp

```diff
@@ -154,11 +160,179 @@ class SFML_WINDOW_API Keyboard
 
         // Deprecated values:
 
-        Dash      = Hyphen,       //!< \deprecated Use Hyphen instead
-        BackSpace = Backspace,    //!< \deprecated Use Backspace instead
-        BackSlash = Backslash,    //!< \deprecated Use Backslash instead
-        SemiColon = Semicolon,    //!< \deprecated Use Semicolon instead
-        Return    = Enter         //!< \deprecated Use Enter instead
+        Dash      = Hyphen,    //!< \deprecated Use Hyphen instead
+        BackSpace = Backspace, //!< \deprecated Use Backspace instead
+        BackSlash = Backslash, //!< \deprecated Use Backslash instead
+        SemiColon = Semicolon, //!< \deprecated Use Semicolon instead
+        Return    = Enter,     //!< \deprecated Use Enter instead
+        Quote     = Apostrophe //!< \deprecated Use Apostrophe instead
+    };
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Scancodes
```

Comment:
One more question to align across the whole code base:

- Scancode vs ScanCode in technical names?
- scancode vs scan code in documentation terms?

Reply:
I think it's an established enough term to use a single word for it (see also https://en.wikipedia.org/wiki/Scancode, which mentions the alternative "scan code", but uses "scancode" throughout the article). It might help splitting visually when the user has longer identifiers: `mapKeyboardScancodes` vs. `mapKeyBoardScanCodes` 😉 

[SDL](https://wiki.libsdl.org/SDL_Scancode), [GLFW](https://www.glfw.org/docs/latest/input_guide.html#input_key) and [Godot](https://docs.godotengine.org/en/3.3/classes/class_inputeventkey.html) also use `Scancode`.

So I tend to **`Scancode` in identifiers and "scancode" in text.**